core-primitives/stf-executor/src/executor.rs

```diff
@@ -69,7 +69,7 @@ where
 	fn execute_trusted_call_on_stf<PH, E>(
```

Comment:
I kept `Call` in all of these namings, because that's what they still are. They're either `indirect_calls` or `direct_calls`, but never getters. So in order to make that clearer, I thought I keep the previous name. Ideally, we would refactor the `TrustedOperation` type to have an additional hierarchy level:
`TrustedOperation` -> `TrustedCall` -> `IndirectTrustedCall` /  `DirectTrustedCall`

Reply:
I guess this will need to be changed anyway once we have two separate pools.

Reply:
Yes, I also think there will be more refactorings necessary for this in the future anyway